ui-testing-core/src/main/java/com/avito/android/test/espresso/action/recycler/RecyclerViewScroll.kt

```diff
@@ -0,0 +1,196 @@
+package com.avito.android.test.espresso.action.recycler
+
+import android.support.test.espresso.PerformException
+import android.support.test.espresso.UiController
+import android.support.test.espresso.ViewAction
+import android.support.test.espresso.contrib.RecyclerViewActions
+import android.support.test.espresso.intent.Checks
+import android.support.test.espresso.matcher.ViewMatchers
+import android.support.test.espresso.util.HumanReadables
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import com.avito.android.test.espresso.action.scroll.scrollToScrollableParentCenterPosition
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+
+class ScrollToPositionViewAction(private val position: Int) : ViewAction {
+
+    override fun getConstraints(): Matcher<View> {
+        return Matchers.allOf(
+            ViewMatchers.isAssignableFrom(RecyclerView::class.java),
+            ViewMatchers.isDisplayed()
+        )
+    }
+
+    override fun getDescription(): String = "scroll RecyclerView to position: $position"
+
+    override fun perform(uiController: UiController, view: View) {
+        val recyclerView = view as RecyclerView
+
+        recyclerView.scrollToPositionToCenter(
```

Comment:
@danazarkov Now I'm using common logic for scrolling view inside RecyclerView item to center of parent